src/__tests__/setup.ts

```diff
@@ -10,3 +10,5 @@ afterEach(() => {
     });
   }
 });
+
+jest.setTimeout(15000);
```

Comment:
The timeout issue was introduced a few Next versions ago since Next.js bootstraps Webpack at configuration loading time. In order to prevent this we temporarily mocked out the module responsible for loading webpack in:  `src/nextConfig.ts`.

```
  const configUtils = require('next/dist/next-server/server/config-utils');
```
should become:
```
const configUtils = require('next/dist/server/config-utils');
```
I'd also ask if you could add a next-page-tester console warning in the catch clause to help detecting future internal changes.

Otherwise we might consider dropping Next.js config loading and implement our own without webpack. We might also double check to see whether configuration loading module now provides a way to skip webpack